.github/workflows/coverage-linux.yml

```diff
@@ -1,18 +1,26 @@
-# TODO(bcoe): add similar job for Windows coverage.
 name: coverage-linux
 
 on:
   pull_request:
+    paths-ignore:
+      - 'doc/**'
```

Comment:
```suggestion
      - '.github/**'
      - 'doc/**'
```

Reply:
But we would want to run this workflow if the workflow itself (`.github/workflows/coverage-linux.yml`) was modified.